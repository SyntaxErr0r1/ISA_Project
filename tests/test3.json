{
    "args": "FEEDURL -T -a -u",
    "feedUrl": [
        "en.wikipedia.org.xml"
    ],
    "returnCode": 0
}