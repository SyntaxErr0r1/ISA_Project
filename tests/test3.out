*** Wikipedia  - Recent changes [en] ***
Battery Ventures
URL: https://en.wikipedia.org/w/index.php?title=Battery_Ventures&diff=1121170609&oldid=1121170545
Aktualizace: Thu, 10 Nov 2022 21:52:44 GMT

User:Jonesey95/sandbox
URL: https://en.wikipedia.org/w/index.php?title=User:Jonesey95/sandbox&diff=1121170608&oldid=1119619673
Aktualizace: Thu, 10 Nov 2022 21:52:44 GMT

Eastern Mountain Sports
URL: https://en.wikipedia.org/w/index.php?title=Eastern_Mountain_Sports&diff=1121170605&oldid=1107343325
Aktualizace: Thu, 10 Nov 2022 21:52:43 GMT